docs/cimpl6Authoring.md

```diff
@@ -98,69 +96,80 @@ Decide if you need to import any namespaces (e.g. OBF).
 
-[CIMPL Tooling Reference Guide](cimpl6ToolingReference.md)
+[CIMPL Tooling Reference Manual](cimpl6ToolingReference.md#front-matter-files)
 
 ## FHIR Examples
 
 A FHIR IG can be generated without examples, however it is recommended to include examples.
 
-The FHIR IG generation process requires that an examples folder/directory be defined in the Configuration file. The folder may be empty.
+The examples file location is declared in the Configuration file.
 
-[CIMPL Tooling Reference Guide](cimpl6ToolingReference.md)
+[CIMPL Tooling Reference Manual](cimpl6ToolingReference.md#fhir-examples)
 
 ## Compile the CIMPL Model
 
-This step runs a syntax check on the Class, Map and Value Set files and prepares files for IG generation.
+This step runs a syntax check on the Class, Map and ValueSet files and prepares files for IG generation.
 
-Errors may be issued from this step.
+Errors may be issued from this step.  
 
-[CIMPL Language Reference Guide](cimpl6LanguageReference.md)
+[CIMPL Tooling Reference Manual](cimpl6ToolingReference.md#executing-shr-cli)
 
 ## Publish the FHIR IG
 
 The final step in the IG creation process is to run the **[FHIR IG Publisher](http://wiki.hl7.org/index.php?title=IG_Publisher_Documentation)**. This tool is maintained and owned by HL7 FHIR.
 
-By default, the FHIR IG Publisher performs validation checks on the  StructureDefinition of specified FHIR profiles, value sets, and examples which reference any base resources or FHIR profiles.  An output of these checks are found in the CIMPL output, *qa.html*.
+By default, the FHIR IG Publisher performs validation checks on the **StructureDefinition** of specified FHIR **profiles**, **value sets**, and **examples** which reference any FHIR **resources** or **profiles**.  Output of these checks are found in the CIMPL output, _qa.html_.
 
-[CIMPL Language Reference Guide](cimpl6LanguageReference.md)
+[CIMPL Tooling Reference Manual](cimpl6ToolingReference.md#creating-the-implementation-guide)
 
 ## Support
 
 Questions on using CIMPL and its toolchain (SHR-CLI) can be addressed on the HL7 Zulip chat channel [#cimpl](https://chat.fhir.org/#streams/197290/cimpl)
 
 If you find an issue you can't resolve, or have a question, report it on one of two JIRA projects:
 
-* Issues related to running the CIMPL SHR-CLI compiler, configuration files, or generating the FHIR Implementation Guide (IG): https://standardhealthrecord.atlassian.net/projects/CIMPL/issues
-* Issues related to CIMPL base classes (Objective FHIR): https://standardhealthrecord.atlassian.net/projects/SHRM/issues
+* Issues related to running the CIMPL SHR-CLI compiler, configuration files, or generating the FHIR IG: https://standardhealthrecord.atlassian.net/projects/CIMPL/issues
+* Issues related to CIMPL base classes (OBF): https://standardhealthrecord.atlassian.net/projects/SHRM/issues
+
+# Appendix A - Document Conventions
+
```

Comment:
We should use the same Document Conventions across all documents in the suite. The Language Reference has the updated version:

# Appendix - Document Conventions

| Style | Explanation | Example |
|:----------|:---------|:---------|
| **Bold**  | A FHIR term or resource | **Profile** |
| `Code` | A CIMPL term, phrase, example, or command | `CodeSystem: LNC = http://loinc.org` |
| <code><i>Italics</i> appearing in a code block | Indicates an item that should be substituted | <code>Value only <i>datatype</i></code> |
| _Italics_ | A file name (also used for general emphasis in text) | _obf-action.txt_ |
| _Italics with **bold** highlight_ | Indicates a substring in the file name that should be substituted | _ig-**myigname**-config.json_ |
| Leading Capitalization | CIMPL keywords or references that are capitalized; specific instances of FHIR artifacts | The `Grammar` keyword |
| **Note:** | Something to keep in mind about the current topic | **Note:** Value Set names must begin with an uppercase letter. |

Reply:
Agree - the document conventions are evolving. Will need a final task to align them all.

Reply:
Yes, and the first evolution is to get rid of bolded words that come from the _general_ FHIR vocabulary. I agree with bolding resource _names_ but not words like "extension", "profile" and "resource".

Reply:
Lets run with it for now and see if we get feedback.